.github/actions/run-python-test-set/action.yml

```diff
@@ -73,13 +73,22 @@ runs:
       shell: bash -euxo pipefail {0}
       run: ./scripts/pysync
 
+    - name: Download compatibility snapshot for Postgres 14
+      uses: ./.github/actions/download
+      with:
+        name: compatibility-snapshot-${{ inputs.build_type }}-pg14
```

Comment:
We would add pg15 eventually, might be worth to extract it to a parameter with the default value already.
Seems that there are more similar places in the workflow yml, so might be a good separate task.